tests/conftest.py

```diff
@@ -27,6 +28,34 @@ def temp_solcx_path(monkeypatch):
         shutil.rmtree(solcx_install_path, ignore_errors=True)
 
 
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
To prevent having to download every time, it might be nice to support disabling this.

Thoughts?

And we have a note in the contributing guide to set `APE_SOLIDITY_TESTS_LITE_MODE=true` or something and it won't download if it doesn't need to

Reply:
Yeah, I think maybe something along the lines of like `APE_SOLIDITY_USE_SYSTEM_SOLC` since the env would basically control whether or not we swap the default installation path

I updated in 0f0ead8 to add the env var and update `CONTRIBUTING.md`

Reply:
Oops, extra `not` in my commit - just force pushed to 9710c9b to correct